tools/pipelines/build-api-markdown-documenter.yml

```diff
@@ -32,6 +32,7 @@ trigger:
   paths:
     include:
     - .prettierignore
+    - common/build/build-common/tsconfig.node16.json # Required since this package references it by file path in its tsconfig
```

Comment:
and that one refers to common/build/build-common/tsconfig.base.json.
If you want just one use base and set "module": "Node16" locally.
("moduleResolution": "Node16" is implied - both are in the common file just as a precaution if there are multiple files inherited.)

Reply:
I went ahead and just added the entire `build-common` directory as a trigger. Single-file references seem to brittle, and I had to add another one anyways.